Utils/AboutUtils.swift

```diff
@@ -11,6 +11,10 @@ struct AboutUtils {
         return getAboutComponent(url) == "home"
     }
 
+    static func isAboutURL(url: NSURL?) -> Bool {
+        return getAboutComponent(url) == "sessionrestore" || getAboutComponent(url) == "home"
```

Comment:
This isn't really "is about URL", but rather "is a sessionrestore or home URL". I'd change this to return `true` for *any* about URL (i.e., `return getAboutComponent(url) != nil`).